packages/@react-spectrum/button/stories/Button.stories.tsx

```diff
@@ -122,3 +127,11 @@ function render<T extends ElementType = 'button'>(props: SpectrumButtonProps<T>
     </Flex>
   );
 }
+
+function Example() {
+  let [show, setShow] = React.useState(false);
+
+  return (
+    <Button variant="cta" UNSAFE_style={show && {background: 'red', userSelect: 'text'}} onPressStart={() => setTimeout(() => setShow(true), 3000)}>Press and hold</Button>
```

Comment:
seems like a pretty easy one to turn into a unit test instead? then we don't need the extra story?